### PR TITLE
[Feature]: Issue & PR Template Added

### LINKED ISSUE #14: [Feature] - Add Issue and PR Template
**Description:** This Repo doesn't have an issue or a PR template

**Expected Behavior:** The Repository should have a Issue as well as a PR template

**For Reference:** 

- PR Template: https://github.com/layer5io/layer5/blob/master/.github/PULL_REQUEST_TEMPLATE.md
- Issue Template: https://github.com/layer5io/layer5/tree/master/.github/ISSUE_TEMPLATE
- Config File: https://github.com/layer5io/layer5/blob/master/.github/config.yml

### PR DESCRIPTION
**Pull request by aminoxix**
_Signed-off-by: amino19 <anshumaankrprasad76@gmail.com>_

As suggested, by @adithyaakrishna, the feature trigger from this PR.

This PR fixes #14 

**Note:**
Hope this changes feature the request, needs review!

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/community_member_profile.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+name: 👤 Community Member Profile
```
Comment (adithyaakrishna) [inline .github/ISSUE_TEMPLATE/community_member_profile.md]: @amino19 This file is not needed 👀

File: .github/ISSUE_TEMPLATE/engagement.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+name: 👨‍🏫 Engagement Request
```
Comment (adithyaakrishna) [inline .github/ISSUE_TEMPLATE/engagement.md]: @amino19 This file is not needed too 👀

File: .github/ISSUE_TEMPLATE/events.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+name: 🎪 📅 Layer5 Event Update
```
Comment (adithyaakrishna) [inline .github/ISSUE_TEMPLATE/events.md]: @amino19 This file is not needed 👀

File: .github/ISSUE_TEMPLATE/landscape.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+name: 🏞 Service Mesh Landscape Update
```
Comment (adithyaakrishna) [inline .github/ISSUE_TEMPLATE/landscape.md]: @amino19 This file is not needed 👀

File: .github/ISSUE_TEMPLATE/learn_layer5.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+name: 📔 Learn Layer5 Site Update
```
Comment (adithyaakrishna) [inline .github/ISSUE_TEMPLATE/learn_layer5.md]: @amino19 This file is not needed too 👀

File: .github/ISSUE_TEMPLATE/news.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
+---
+name: 📰 Layer5 News Update
```
Comment (adithyaakrishna) [inline .github/ISSUE_TEMPLATE/news.md]: @amino19 This file is not needed also👀

**Review by adithyaakrishna — CHANGES_REQUESTED** [review]
Yo @amino19 Please do make the changes as suggested and double check the links as Meshery was moved from layer5io GitHub Org. to leshery Org.

**Comment by aminoxix** [comment]
@adithyaakrishna : Thanks! :slightly_smiling_face: 
I think here's no link relates to meshery/ org., this itself in Layer5.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+[Optional] **Additional Context**
+<!-- Add any other context about the problem here. -->
+
+---
+**Contributor Resources**
```
Comment (adithyaakrishna) [inline .github/ISSUE_TEMPLATE/bug_report.md:36]: @amino19 We don't need this specific section except the line below,

- `See [contributing instructions(https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)`

File: .github/ISSUE_TEMPLATE/chore_task.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+[Optional] **Alternatives**
+<!-- A brief description of any alternative solutions or features you've considered. -->
+
+[Optional] **Additional context**
+<!-- Add any other context or screenshots about the chore or task here. -->
+
+---
+**Contributor Resources**
+
+The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
```
Comment (adithyaakrishna) [inline .github/ISSUE_TEMPLATE/chore_task.md]: @amino19 Refer this, https://github.com/layer5io/gowrk2/pull/15/files#r663357864

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+
+---
+**Screenshots / Mockups**
+<!-- Add any other context or screenshots about the feature request here. -->
+
+**Alternatives**
+<!-- A brief description of any alternative solutions or features you've considered. -->
+
+---
+**Contributor Resources**
```
Comment (adithyaakrishna) [inline .github/ISSUE_TEMPLATE/feature_request.md:22]: @amino19 Refer this, https://github.com/layer5io/gowrk2/pull/15/files#r663357864

**Review by adithyaakrishna — CHANGES_REQUESTED** [review]
@amino19 Last set of changes, everything else LGTM 🎉

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+
+The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
```
Comment (adithyaakrishna) [inline .github/ISSUE_TEMPLATE/bug_report.md]: @amino Could you please remove this line also? 👀

File: .github/ISSUE_TEMPLATE/chore_task.md [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
+- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.
```
Comment (adithyaakrishna) [inline .github/ISSUE_TEMPLATE/chore_task.md]: @amino Could you please remove this line also? 👀

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+
+The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
```
Comment (adithyaakrishna) [inline .github/ISSUE_TEMPLATE/feature_request.md]: @amino Could you please remove this line also? 👀

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 13]
```diff
@@ -0,0 +1,26 @@
...
+- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
+- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.
```
Comment (adithyaakrishna) [inline .github/ISSUE_TEMPLATE/feature_request.md]: @amino Could you please remove this line too? 👀

**Review by adithyaakrishna — CHANGES_REQUESTED** [review]
The lines are still remaining, have tagged them above. Lmk once the changes are done. Will review it 💯

**Comment by aminoxix** [comment]
@adithyaakrishna : Done! :100:

**Review by adithyaakrishna — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 7 above)
Comment (leecalcote) [reply .github/ISSUE_TEMPLATE/bug_report.md:36]: Good catch!

**Comment by adithyaakrishna** [comment]
Hey @amino19 , the PR Approval has been notified to the Maintainers, please do wait for sometime as they might be a bit busy  😶

**Comment by aminoxix** [comment]
@adithyaakrishna : Oh yeah, I got, Thanks for this info.! :slightly_smiling_face:

**Comment by leecalcote** [comment]
Good work. 🥇

**Pull request merged**